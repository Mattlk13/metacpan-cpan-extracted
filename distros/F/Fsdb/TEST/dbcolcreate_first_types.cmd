prog='dbcolcreate'
args='-f -e 1.0 newtype:d'
cmp='diff -c -b '

