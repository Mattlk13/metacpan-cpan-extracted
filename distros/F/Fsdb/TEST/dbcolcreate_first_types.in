#h test number:l
a 1
b 2
