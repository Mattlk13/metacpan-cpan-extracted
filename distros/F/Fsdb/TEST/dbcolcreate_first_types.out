#fsdb newtype:d test number:l
1.0	a	1
1.0	b	2
#   | dbcolcreate -f -e 1.0 newtype:d
